exchanges/ticker/ticker_test.go

```diff
@@ -15,7 +15,7 @@ import (
 )
 
 func TestMain(m *testing.M) {
-	err := dispatch.Start(1)
+	err := dispatch.Start(1, 0)
```

Comment:
Can set this to dispatch.DefaultJobsLimit, so the check < 0 won't trigger a warning and that the value is set to defaults anyway